modules/ROOT/pages/dataweave-formats-ndjson.adoc

```diff
@@ -144,11 +144,13 @@ This format accepts properties that provide instructions for writing output data
 [cols="1,1,1,3a", options="header"]
 |===
 |Parameter |Type |Default|Description
-|`bufferSize`  |`Number`|`8192`|Size of the buffer writer.
-|`deferred`  |`Boolean`|`false`|Generates the output as a data stream when set to `true`, and defers the script's execution until consumed.
+|`bufferSize`  |`Number`|`8192`|Size of the buffer writer. The value must be greater than 8.
+
+|`deferred`  |`Boolean`|`false`|Generates the output as a data stream when set to `true`, and defers the script's execution until the generated content is consumed.
+
 
 Valid values are `true` or `false`.
-|`encoding`  |`String`|`'UTF-8'`|The character set to use for the output, such as UTF-8.
+|`encoding`  |`String`|`'UTF-8'`|Encoding that the writer uses for output. Defaults to "UTF-8".
```

Comment:
TODO: Conform here and in source with other descriptions of this same property.

Reply:
Double check the default and whether single or double quotes can be used.

Reply:
TODO: check with team

Reply:
added reminder to next PR on this https://github.com/mulesoft/docs-dataweave/pull/225/files